brands/shop/bakery.json

```diff
@@ -101,6 +101,17 @@
       "shop": "bakery"
     }
   },
+  "shop/bakery|BreadTalk": {
+    "countryCodes": ["sg"],
+    "tags": {
+      "amenity": "cafe",
```

Comment:
oh I just noticed this extra `"amenity": "cafe"` still here so I removed it in eeeff003b